proto/babylon/btcstaking/v1/query.proto

```diff
@@ -69,6 +74,17 @@ message QueryParamsByVersionResponse {
   Params params = 1 [(gogoproto.nullable) = false];
 }
 
+// QueryParamsByBTCHeightRequest is request type for the Query/ParamsByBTCHeight RPC method.
+message QueryParamsByBTCHeightRequest {
+  uint32 btc_height = 1;
+}
+
+// QueryParamsByBTCHeightResponse is response type for the Query/QueryParamsByBTCHeightResponse RPC method.
+message QueryParamsByBTCHeightResponse {
+  // params holds all the parameters of this module.
+  Params params = 1 [(gogoproto.nullable) = false];
```

Comment:
would it be helpful to also have version number here?

Reply:
probably 👍 good idea